lib/pages/profile_page.dart

```diff
@@ -9,31 +10,111 @@ class ProfilePage extends StatefulWidget {
   }
 }
 
-class _ProfilePageState extends State<ProfilePage> {
+class _ProfilePageState
+    extends State<ProfilePage> {
   double? _deviceHeight;
   double? _deviceWidth;
```

Comment:
Again, not something this PR did but _deviceWidth isn't used as far as I can tell as it is redefined in the build